cmd/buildah/images.go

```diff
@@ -109,30 +133,17 @@ func imagesCmd(c *cli.Context) error {
 		return errors.Errorf("quiet and format are mutually exclusive")
 	}
 
-	quiet := c.Bool("quiet")
-	truncate := !c.Bool("no-trunc")
-	digests := c.Bool("digests")
-	hasTemplate := c.IsSet("format")
+	opts := imageOptions{
+		all:        c.Bool("all"),
+		digests:    c.Bool("digests"),
+		format:     c.String("format"),
+		json:       c.Bool("json"),
+		noHeading:  c.Bool("noheading"),
+		noTruncate: c.Bool("notruncate"),
```

Comment:
I think I'd rather leave this as truncate and still do the !c.Bool("notruncate") here.  Makes the code a little more understandable down below.

Reply:
updated, PTAL.